Data/Sys/GameSettings/RMC.ini

```diff
@@ -19,6 +19,11 @@ EmulationIssues =
 
 [Video_Hacks]
 EFBEmulateFormatChanges = True
+EFBToTextureEnable = False # Correct lens flare
```

Comment:
Hm, this one will be a major slowdown. Are you sure this should be enforced for everyone? Maybe just a "EmulationIssues" comment.